vxquery-cli/src/main/java/org/apache/vxquery/cli/VXQuery.java

```diff
@@ -120,200 +94,236 @@ public static void main(String[] args) throws Exception {
             parser.printUsage(System.err);
             return;
         }
-        if (opts.arguments.isEmpty()) {
+
+        if (opts.xqFiles.isEmpty()) {
             parser.printUsage(System.err);
             return;
         }
+
         VXQuery vxq = new VXQuery(opts);
-        vxq.execute();
-        // if -timing argument passed, show the starting and ending times
-        if (opts.timing) {
```

Comment:
Yes. That's one thing I mentioned in the email sent to you (4-5 days back). Since we are using 2 requests to execute and fetch results for a given query, we are missing the actual time spent on executing the query. But in the CLI, we have a small chance since we are waiting for the result as soon as we submit the query to be processed. Based on that assumption, I can add the missing timing metrics if you think it is ok.